modules/ROOT/pages/distributed-file-polling.adoc

```diff
@@ -0,0 +1,23 @@
+= Distributed File Polling
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: file polling, cluster
+
+*_Enterprise Edition_*
+
+Some endpoints, such as the File Endpoint or FTP Endpoint, regularly poll directories and read certain files as they are created in the directories polled. These files may reside on a remote file system, including file systems of nodes belonging to a Mule High Availability (HA) Cluster.
+
+Up to Mule 3.3.x, all file poll requests sent to a cluster were channeled through a single node in that cluster. This ensured file consistency by avoiding the possibility of file system sync issues, but had the disadvantage of introducing a bottleneck, reducing the performance of the cluster.
```

Comment:
I think we should remove the comparison with Mule 3.3.x since it no longer makes sense

Reply:
Updated